### PR TITLE
Add type stubs for google.protobuf.service

### PR DESCRIPTION
**Pull request by nipunn1313**
I used stubgen and manually filled in the rest.

We use these types internally at Dropbox, as well as on mypy-protobuf (also used at Dropbox).
For context, mypy-protobuf currently has open issue https://github.com/dropbox/mypy-protobuf/issues/25, with a (hacky) workaround (https://github.com/dropbox/mypy-protobuf/commit/0df796bcd2296df91a511dd994bd476e04bc6f93) while we wait for these annotations to come through.

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Can you fix the CI errors? (After you do that you may get other errors from pytype, which can be ignored for now.)

File: third_party/2and3/google/protobuf/service.pyi [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Stubs for google.protobuf.service (Python 3.6)
```
Comment (JelleZijlstra) [inline third_party/2and3/google/protobuf/service.pyi]: These comments can be removed; they don't add a lot of information.

**Comment by nipunn1313** [comment]
I'll get around to this - I've been out of town for a while - but will do when I get back. Sorry for the delay.

**Comment by nipunn1313** [comment]
Cool! Tests pass - I updated for the CI errors.
Thanks

**Comment by nipunn1313** [comment]
Thanks @gvanrossum and @JelleZijlstra ! ❤️

**Pull request merged**